spec/JSON-RPC.md

```diff
@@ -1931,6 +1932,63 @@ curl \
 
 [Back to **List of methods**](#list-of-methods)
 
+## mempool_getMinimumFees
```

Comment:
The RPC method seems to general.
Since we have two types of minimum fee, we need a verbose RPC name.